interfaces/services/consensus_context.proto

```diff
@@ -90,3 +91,13 @@ message RequestCommitteeOutput {
     repeated primitives.node_address node_addresses = 1;
     repeated primitives.bls1_public_key node_random_seed_public_keys = 2;
 }
+
+message RequestBlockProofCommitteeInput {
+    primitives.timestamp_seconds prev_block_reference_time = 1;
+    uint32 max_committee_size = 2;
```

Comment:
if max_committee_size is not in use let's remove instead of keeping a place holder

Reply:
I'll drop it